data/coalition/coalition missions.txt

```diff
@@ -1393,3 +1393,57 @@ mission "Arachi Cartoonist 2"
+			`	"Greetings, Captain. Interested in participating in a cultural celebration, would you be?" He asks.`
+			choice
+				`	"What sort of celebration?"`
+				`	"Sorry, I'm busy right now."`
+					decline
+			`	"A celebration of Saryd folklore, it is," he explains. "Though for a long time among the stars we've lived, timeless classics from before our spacefaring age, we still celebrate. A set of theatrical plays about some of them, today we are hosting. Watch them for free, anyone can."`
+			choice
+				`	"I've always got time for a story, lead the way."`
+				`	"I'm not much of a fan of plays, I'll pass."`
+					decline
+			`	You follow him outside the spaceport city, and deep into a nearby forest. On the way there, several other Saryds in ancient garbs are guiding others who were also intrigued by the offer.`
+			`	You arrive at a clearing with three different paths to take, and you can make out a faint light deep within each of them. After a few minutes it seems as if all the Saryds involved in the celebration have arrived, as they grab three caskets from behind a bush, and lay them before you and the others they brought here.`
+			`	"Now friends, choose your path, you must," one of the Saryds says, and many from the crowd grab small objects from the caskets, seemingly already familiar with this. You move forward to look into the caskets, and see that each one has different things inside: the left one is full of seeds, the middle one full of seashells, and the right one full of feathers.`
+			choice
+				`	(Grab a seed.)`
+					goto seed
+				`	(Grab a seashell.)`
+					goto pearl
+				`	(Grab a feather.)`
+			`	Once you have the feather in your hands, the organizers tell you to go through the path on the right, and you do so.`
+			`	The light you had seen turns out to be a bonfire, and you find a spot to sit on one of the logs close to it. You keep your eyes on the wooden stage in front of you, about a third of the size of human opera stages. Several lights shine onto it, and as you look up you can only barely see some spotlights high up in the trees. After the audience is all here, the curtains open to reveal many small figures made out of what looks like paper in a style that loosely resembles origami. They are supported either by thin wooden poles from below, or strings from above. There are figures close to where the curtains were, and others further into the stage, the various layers granting depth to the scenery. After a brief moment, the show begins.`
+			`	Entering a clearing, a Saryd huntress with gray hair stalks a large bird resting on a rock. She moves on the borders of the clearing, hiding behind bushes and small trees. She looks for the best spot from which to snipe at the bird, her bow and arrow ready in her hands. It's only when she is nearing a large entrance to the clearing, on the opposite side of where she entered, that she finds the spot, and takes aim.`
```

Comment:
I think this paragraph has the same issue the whale in the seashell story had. When I read that a Saryd was hunting a bird, I picture a hawk or some other obviously Earth animal. Maybe change it so that it's clear that the Saryd is hunting some bird-like alien animal, and not a literal bird from Earth.

Reply:
I'd actually have to disagree with this. A "bird" is a generic classification, where a "whale" is a specific creature.

Reply:
Hmm, but 'bird' is a Earth classification of animal. Alien animals wouldn't fit into Earth biology. The most that they could do is somewhat resemble birds. 
Maybe I'm just being really nitpicky. :P

Reply:
We could just start calling everything "bird-like", "beetle-like", or "spider like" but it would get cumbersome, and probably annoying to read, pretty quickly. While I can understand arguments for not calling alien creatures by specific human animal names, I think when it general classification suchs as "bird", "fish", "insect" it's perfectly exceptable to just go with it. The average player isn't going to want a biology lesson in the story, those are saved for logbook entries.

Reply:
May I suggest "avian" or "avian creature". Not as short as "bird", but perhaps adds some variation and leaves a bit more room for interpretation.

Reply:
I don't think it's worth describing the bird, in Coalition planet descriptions there are cases of "shoals of fish" or "spiders so large that their webs are strung from one tree to another," so it's not like every single instance of an animal being mentioned needs to be accompanied by a description to show how it's not an exact copy of an Earth animal, so I'll just leave this one as it is.

Reply:
Fair.